QuowMinimap.lua

```diff
@@ -262,12 +262,15 @@ if (string.sub(iMushclientVersion, -4) == "-pre") then
 end
 iMushclientVersion = tonumber(iMushclientVersion)
 
+bIsWindows = GetInfo(268) < 100
```

Comment:
In MUSHclient, `GetInfo(268)` returns one of the following:
- 0: Win32
- 1: Windows 95
- 2: All other Windows versions

SmushClient adds two more return values:
- 100: MacOS
- 200: Linux